theme/templates/resource-landing-page/subject.html

```diff
@@ -49,4 +52,4 @@ <h3>Subject Keywords</h3>
     </div>
 </div>
 
-<script type="text/javascript" src="{{ STATIC_URL }}js/hs-vue/subjet-keywords.js"></script>
```

Comment:
very minor but did you mean to name the file subject? The 'c' is missing.

Reply:
I did. Will fix it.